nix/scripts/node_modules.sh

```diff
@@ -61,6 +61,8 @@ findFilesNewerThan() {
       -not -ipath "*/*android/build/*" -prune \
       -not -ipath "*/xcuserdata/*" -prune \
       -not -ipath "*/scripts/.packager.env" \
+      -not -path "*/node_modules/react-native/ReactCommon/react/renderer/components/rncore/*" \
+      -not -path "*/node_modules/react-native-config/ios/ReactNativeConfig/GeneratedDotEnv.m" \
```

Comment:
reason for `GeneratedDotEnv.m` change seems to be due to https://github.com/lugg/react-native-config package.
I had also observed this during my investigations here -> https://github.com/status-im/status-mobile/issues/15911#issuecomment-1549537754